APIComparer.Contracts/CompareNugetPackage.cs

```diff
@@ -1,20 +1,20 @@
 ﻿// ReSharper disable MemberCanBePrivate.Global
 namespace APIComparer.Contracts
 {
-    using System;
     using NServiceBus;
+    using NuGet;
 
     public class CompareNugetPackage : ICommand
     {
-        public CompareNugetPackage(string packageId, Version leftVersion, Version rightVersion)
+        public CompareNugetPackage(string packageId, SemanticVersion leftVersion, SemanticVersion rightVersion)
         {
             PackageId = packageId;
             RightVersion = rightVersion;
             LeftVersion = leftVersion;
         }
 
-        public string PackageId { get; private set; }
-        public Version LeftVersion { get; private set; }
-        public Version RightVersion { get; private set; }
+        public string PackageId { get; set; }
+        public SemanticVersion LeftVersion { get; set; }
```

Comment:
Not sure its a good idea to tie the message contract to a nuget type?

How about just a string for now?